server/modules/apiCallToAddSongsToDB.js

```diff
@@ -10,13 +10,16 @@ const apiCallToAddSongsToDB = function (req, res) {
 
   db.query(query, [artist_name, song_name, spotify_song_id])
     .then(() => {
-      res.status(201).send({ result: "Song added successfully, enjoy!" });
+      res.status(201).send({
+        result: "success",
+        message: `${song_name} by ${artist_name} added successfully!`,
```

Comment:
Nice change! I like how the song name and artist is in the response message. It'll let users confirm which song is added